openshift-kube-apiserver/admission/autoscaling/managementcpusoverride/admission.go

```diff
@@ -0,0 +1,236 @@
+package managementcpusoverride
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"k8s.io/apiserver/pkg/admission"
+	"k8s.io/apiserver/pkg/admission/initializer"
+	"k8s.io/client-go/informers"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/klog/v2"
+	coreapi "k8s.io/kubernetes/pkg/apis/core"
+	"k8s.io/kubernetes/pkg/kubelet/cm"
+)
+
+const (
+	PluginName                 = "autoscaling.openshift.io/ManagementCPUsOverride"
+	podWorkloadLabel           = "workload.openshift.io"
+	managedCapacityLabel       = "workload.openshift.io/cores"
+	containerCPUsAnnotation    = "io.openshift.workload.management.cpu"
+	containerCPUSetAnnotation  = "io.openshift.workload.management.cpuset"
```

Comment:
This isn't required to be set since it is set directly in crio.conf using Machine Config.